cmd/skupper/README.md

```diff
@@ -0,0 +1,98 @@
+# Installing the skupper controller
+
+```
+kubectl create namespace skupper
+kubectl config set-context --current --namespace skupper
+```
+
+Apply the CRDs in the cluster:
+
+```
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_access_grant_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_access_token_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_attached_connector_anchor_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_attached_connector_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_certificate_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_connector_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_link_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_listener_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_router_access_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_secured_access_crd.yaml
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/api/types/crds/skupper_site_crd.yaml
+```
+
+Deploy the Skupper controller in the cluster:
+```
+kubectl apply -f https://raw.githubusercontent.com/skupperproject/skupper/v2/cmd/controller/deploy_cluster_scope.yaml
+```
+
+# Deploy application in two namespaces (or contexts)
+
+E.g.
+
+```
+kubectl create namespace west
+kubectl create deployment frontend --image quay.io/skupper/hello-world-frontend -n west
+```
+
+```
+kubectl create namespace east
+kubectl create deployment backend --image quay.io/skupper/hello-world-backend --replicas 3 -n east
+```
+
+# Create sites
+
+```
+skupper site create site1 --enable-link-access -n west
+```
+
+```
+skupper site create site2  -n east
+```
+
+# Expose backend in east site (create a connector)
+
+```
+skupper connector create backend 8080 --routing-key backend --selector app=backend -n east
+```
+
+# Consume backend in west site (create a listener)
+
+```
+skupper listener create backend 8080 --host backend --routing-key backend -n west
+```
+
+# Link sites
+
+Create a AccessGrant in west site and generate a file with the accessToken:
```

Comment:
Apologies for the complete nit but: consider updating accessToken -> AccessToken?